code/modules/mob/living/carbon/human/species/skeleton_species.dm

```diff
@@ -47,3 +53,16 @@
 		return TRUE
 
 	return ..()
+
+/// Wizard subtype, subtype to allow balancing separately from other skellies
+/datum/species/skeleton/lich
+	name = "Lich"
+	name_plural = "Liches"
+
+/// The most common (and weakest) type, legion corpses and skeleton map spawners are these
+/datum/species/skeleton/brittle
+	name = "Brittle Skeleton"
+	name_plural = "Brittle Skeletons"
+	inherent_traits = list(TRAIT_RESISTHEAT, TRAIT_NOBREATH, TRAIT_RESISTHIGHPRESSURE, TRAIT_RADIMMUNE, TRAIT_PIERCEIMMUNE, TRAIT_NOHUNGER, TRAIT_XENO_IMMUNE)
+	milk_heal_amount = 0.66 //on par with saline-glucose (after averaging that out)
```

Comment:
Major niche review but your code comments are a bit inconsistent, sometimes you format like (// This) and other times like (//this), not like an actual code issue but it comes off as kind of strange

Reply:
I was drunk as hell when I coded this

Reply:
> I was drunk as hell when I coded this

Haha yeah I get it